tests/stp/pubsub/pubsub.csv

```diff
@@ -37,4 +37,10 @@ run,0,0,q,psHandle(upsert;`trade;testtrade),1,,"Update trade table on pubsub"
 true,0,0,q,1~count psHandle(`trade),1,,"Check there is one row in trade table"
 run,0,0,q,neg[psHandle](`.stpps.pubclear;`trade),1,,"Call pubclear funtion"
 true,0,0,q,0~count psHandle(`trade),1,,"Check that trade table has been cleared"
-true,0,0,q,1~.tst.upd,1,,"Check that local UPD function has been called"
\ No newline at end of file
+true,0,0,q,1~.tst.upd,1,,"Check that local UPD function has been called"
+run,0,0,q,upd:{show x:show count y},1,,"Define upd for sub filter tests"
+run,0,0,q,rdbHandle(`.u.sub;`trade;`AAPL`GOOG),1,,"Subscribe to filtered sym list"
+run,0,0,q,rdbHandle".stpps.subrequestfiltered",1,,"Check filtered subscription has passed"
+run,0,0,q,rdbHandle".u.pub[`trade;-10 sublist trade]",1,,"Force publish for the filtered list"
+run,0,0,q,rdbHandle"upd:insert",1,,"Return upd back to default definition"
+true,0,0,q,rdbHandle"(upd)~(insert)",1,,"Check upd has been reset to default"
```

Comment:
I don't think this test is important, I think it's safe to assume that `:` in q works (which is all this is really testing)